src/main/java/org/sagebionetworks/bridge/reporter/helper/BridgeHelper.java

```diff
@@ -51,11 +55,22 @@ public final void setBridgeCredentials(SignIn bridgeCredentials) {
 
     /*
      * Helper method to get all uploads for specified study and date range
+     * Paginated results should be added in one list altogether
      */
     public List<Upload> getUploadsForStudy(String studyId, DateTime startDateTime, DateTime endDateTime)
             throws IOException {
-        return sessionHelper(() -> bridgeClientManager.getClient(ForWorkersApi.class).getUploadsInStudy(studyId,
-                startDateTime, endDateTime).execute().body().getItems());
```

Comment:
Have we verified that we no longer need the sessionHelper? How have you tested this?